storey/aggregations.py

```diff
@@ -19,29 +19,34 @@ class AggregateByKey(Flow):
     Persistence is done via the `WriteToTable` step and based on the Cache object persistence settings.
 
     :param aggregates: List of aggregates to apply for each event.
-    :param table: A Table object to aggregate the data into.
+    :param table: A Table object to aggregate the data into. Alternatively can provide the table's name if a the relevant Table entry exists
```

Comment:
A Table object or name for persistence of aggregations. If a table name is provided, it will be looked up in the context.